tests/utils/deployment_graph.py

```diff
@@ -88,3 +88,34 @@ def completions_response_handler(response):
     assert len(result["choices"]) > 0, "Empty choices in response"
     assert "text" in result["choices"][0], "Missing 'text' in first choice"
     return result["choices"][0]["text"]
+
+
+def metrics_label_handler(response):
```

Comment:
```suggestion
def metrics_handler(response):
```
Could be expanded to check for other metrics related validation